src/stdlib_experimental_io.f90

```diff
@@ -115,6 +195,70 @@ subroutine dsavetxt(filename, d)
 close(s)
 end subroutine
 
+subroutine qsavetxt(filename, d)
+! Saves a 2D array into a textfile.
+!
+! Arguments
+! ---------
+!
+character(len=*), intent(in) :: filename  ! File to save the array to
+real(qp), intent(in) :: d(:,:)           ! The 2D array to save
+!
+! Example
+! -------
+!
+! real(dp) :: data(3, 2)
+! call savetxt("log.txt", data)
+
+integer :: s, i
+open(newunit=s, file=filename, status="replace")
+do i = 1, size(d, 1)
+    write(s, *) d(i, :)
+end do
+close(s)
+end subroutine
+
+
+!PRIVATE
```

Comment:
I assume this does a similar thing as the `!PUBLIC` above.

Reply:
Removed